hw3/datalore.md

```diff
@@ -0,0 +1,2 @@
+This homework is on Datalore.
+https://datalore.jetbrains.com/notebook/RcWnlNjoSOmKL0Jw0MjiKv/Ln8SKjYPUGfbkqaKNJE7lr/
```

Comment:
In 1c, there are actually 2 people who paid the least. Since you did not notice this, write a code that will print this information: it should be able to correctly print the necessary lines even if several people are tied for the first/last place.